mkosi

```diff
@@ -2170,7 +2226,6 @@ def install_arch(args: CommandLineArguments, root: str, do_run_build_script: boo
                 HookDir     = {root}/etc/pacman.d/hooks/
                 HoldPkg     = pacman glibc
                 Architecture = auto
-                UseSyslog
```

Comment:
Shouldn't we keep this around? Without this there are no logs in the journal of what pacman does.

Reply:
pacman logs everything at log level WARNING which I think is spammy and not very useful. It also logs under the pacman identifier which is unintuitive since when looking back later one might confuse it with pacman running on the host system. Finally, with systemd v246, stuff from the systemd binaries invoked by the pacman hooks is correctly logged to stdout by pacman which makes me think that its stdout output is logging enough.